styles.css

```diff
@@ -29,30 +47,204 @@
+  margin: 0;
+  width: 98%;
+  height: 48%;
+  background-color: white;
+  border-radius: 9px;
+  align-self: center;
+}
+
+.imagePlaceHolder {
+  /* This is an add-on for the recent works cards */
+  height: 50%;
+  width: 100%;
+}
+
+#workName {
+  color: #3a4a42;
+}
+
+/* Reusable */
+
+/* Grid Creator */
+
+.centeredGrid {
+  display: grid;
+  justify-content: center;
+  align-content: center;
+  grid-template-columns: 327px;
+  grid-gap: 24px;
+}
+
+.centeredGrid h2 {
   font-weight: 700;
-  font-size: 30px;
+  font-size: 40px;
   text-align: center;
+  justify-content: center;
+  align-content: center;
+  color: #172b4d;
 }
 
-.mainText p:first-child {
+/* Card creator */
+
+.card {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  text-align: center;
+  background-color: #ebf0ee;
+  border-radius: 9px;
+  flex-direction: column;
+  outline: #d0d9d4;
+  outline-width: 3px;
+  border-color: #d0d9d4;
+  align-content: center;
+}
+
+.cardHeader {
+  line-height: 32px;
+  font-weight: 600;
+  font-size: 24px;
+}
+
+/* Tags creator/handler */
+
+.tagsHandler {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+  padding: 0;
+  width: 100%;
+}
+
+.arrow {
+  margin-top: 64px;
+}
+
+/* Button */
+
+.button {
+  width: auto;
+  height: 48px;
+  background-color: #36b37f;
+  border-width: 0;
+  border-radius: 9px;
+  transition-duration: 0.3s;
+  font-weight: 500;
+  font-size: 17px;
+  color: white;
+  justify-self: center;
+  padding-left: 10px;
+  padding-right: 10px;
+  letter-spacing: 0.03em;
+  font-style: normal;
+}
+
+/* Button Methods */
+
+button:hover {
+  box-shadow: 0 8px 16px 0 #36b37f3d;
+}
+
+.button:active {
+  background-color: #008552;
+}
+
+/* Section header formatter */
+
+.sectionHeader h2 {
+  font-weight: 700;
+  font-size: 40px;
+  color: #172b4d;
   margin-bottom: 0;
 }
 
-.secondaryText {
+.sectionHeader h3 {
   font-weight: 400;
+  color: #42526e;
   font-size: 20px;
+}
+
+/* Small vecor formatter */
+
+#indicator {
+  width: 48px;
+  height: 4px;
+  justify-self: center;
+  background: #36b37e;
+  border-radius: 24px;
+  margin-bottom: 62px;
+}
+
+/* About Me Section */
+
+.aboutme {
+  display: grid;
+  justify-content: center;
+  align-content: center;
   text-align: center;
-  margin: 58px 24px 60px 24px;
+  grid-gap: 28px;
+  grid-auto-rows: 352px;
 }
 
-#secondMainSentence {
-  color: #36b37e;
-  margin-top: 0;
+.aboutmeHeader {
+  background-image: url("./src/misc/aboutme-illustration-mobile.png");
+  background-size: contain;
 }
 
-#welcomeText {
+.aboutmeTags {
+  color: #36b37e;
   font-weight: 600;
-  text-decoration: none;
-  color: black;
+  font-size: 12px;
+  flex-wrap: wrap;
+  justify-content: center;
+  list-style-type: none;
+  align-content: center;
+}
+
+.aboutmeTags li {
+  background-color: white;
+  padding: 10px;
+  border-radius: 9px;
+  text-align: center;
+  margin: 5px 10px;
+  margin-right: 1em;
```

Comment:
- [ ] [OPTIONAL] Kindly remove `margin-right` because you are overwriting your last `margin` property.